### DISCUSSION TITLE
Trying Zluda got stuck. elp

### DISCUSSION THREAD

**Discussion by songib** [lshqqytiger/stable-diffusion-webui-directml#491] [Q&A]
`Committer identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'ssssssssssssss.(none)')
The filename, directory name, or volume label syntax is incorrect.
The filename, directory name, or volume label syntax is incorrect.
The filename, directory name, or volume label syntax is incorrect.
venv "\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
WARNING: ZLUDA works best with SD.Next. Please consider migrating to SD.Next.
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.9.3-amd-29-g0bde866e
Commit hash: 0bde866ea77e48f9b57e6843531abbe81361883b
Using ZLUDA in \stable-diffusion-webui-directml\.zluda
*** Error running install.py for extension \stable-diffusion-webui-directml\extensions\canvas-zoom.
*** Command: "\stable-diffusion-webui-directml\venv\Scripts\python.exe" "\stable-diffusion-webui-directml\extensions\canvas-zoom\install.py"
*** Error code: 1
*** stderr: Traceback (most recent call last):
***   File "\stable-diffusion-webui-directml\extensions\canvas-zoom\install.py", line 4, in <module>
***     import gradio
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\__init__.py", line 3, in <module>
***     import gradio.components as components
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\__init__.py", line 1, in <module>
***     from gradio.components.annotated_image import AnnotatedImage
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\annotated_image.py", line 13, in <module>
***     from gradio.components.base import IOComponent, _Keywords
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\base.py", line 20, in <module>
***     from fastapi import UploadFile
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\__init__.py", line 7, in <module>
***     from .applications import FastAPI as FastAPI
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\applications.py", line 15, in <module>
***     from fastapi import routing
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\routing.py", line 22, in <module>
***     from fastapi import params
***   File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\params.py", line 4, in <module>
***     from pydantic.fields import FieldInfo, Undefined
*** ImportError: cannot import name 'Undefined' from 'pydantic.fields' (\stable-diffusion-webui-directml\venv\lib\site-packages\pydantic\fields.py)
Installing sd-webui-controlnet requirement: changing opencv-python version from 4.10.0.84 to 4.8.0
Faceswaplab : Use GPU requirements
Checking faceswaplab requirements
0.006236600000192993
Traceback (most recent call last):
  File "\stable-diffusion-webui-directml\launch.py", line 48, in <module>
    main()
  File "\stable-diffusion-webui-directml\launch.py", line 39, in main
    prepare_environment()
  File "\stable-diffusion-webui-directml\modules\launch_utils.py", line 669, in prepare_environment
    from modules import devices
  File "\stable-diffusion-webui-directml\modules\devices.py", line 6, in <module>
    from modules import errors, shared, npu_specific
  File "\stable-diffusion-webui-directml\modules\shared.py", line 4, in <module>
    import gradio as gr
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\__init__.py", line 3, in <module>
    import gradio.components as components
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\__init__.py", line 1, in <module>
    from gradio.components.annotated_image import AnnotatedImage
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\annotated_image.py", line 13, in <module>
    from gradio.components.base import IOComponent, _Keywords
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\gradio\components\base.py", line 20, in <module>
    from fastapi import UploadFile
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\__init__.py", line 7, in <module>
    from .applications import FastAPI as FastAPI
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\applications.py", line 15, in <module>
    from fastapi import routing
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\routing.py", line 22, in <module>
    from fastapi import params
  File "\stable-diffusion-webui-directml\venv\lib\site-packages\fastapi\params.py", line 4, in <module>
    from pydantic.fields import FieldInfo, Undefined
ImportError: cannot import name 'Undefined' from 'pydantic.fields' (\stable-diffusion-webui-directml\venv\lib\site-packages\pydantic\fields.py)
Press any key to continue . . . `

Any tutorials are appreciated. and I'm on 5700 series.
only use this command set COMMANDLINE_ARGS= --use-zluda
atm I follow this one [(https://github.com/vladmandic/automatic/wiki/ZLUDA)]
and using torch CPU for some reason.

**Comment by CS1o** [comment]
Try using my Guides if you still need help.
https://github.com/CS1o/Stable-Diffusion-Info/wiki/Webui-Installation-Guides
